src/pybamm/util.py

```diff
@@ -115,7 +148,6 @@ def search(self, keys: str | list[str], print_values: bool = False):
         the best matches are printed.
 
         Parameters
-        ----------
```

Comment:
Removing this dash messes up the formatting:
![image](https://github.com/user-attachments/assets/fedaa74f-f376-40e3-b821-b2c64252ab3f)

Reply:
Sure, I have made the changes.

Reply:
Can you push that change, the dashes are still removed. They need to be replaced in both spots

Reply:
[f262534](https://github.com/pybamm-team/PyBaMM/pull/4846/commits/f26253466b037a8896d5af85dd1fd26d3888f35d) Please verify.